boa/src/syntax/parser/statement/declaration/hoistable/tests.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Why is this empty / exists?

Reply:
I think this highlights we don't actually have any tests specifically targeting hoistable statement decl except those for function decl / async function decl